File: src/main/java/org/example/kalkulationsprogramm/repository/KrankenkasseRepository.java
package org.example.kalkulationsprogramm.repository;

import org.example.kalkulationsprogramm.domain.Krankenkasse;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface KrankenkasseRepository extends JpaRepository<Krankenkasse, Long> {
    List<Krankenkasse> findAllByOrderByNameAsc();
    List<Krankenkasse> findByAktivTrueOrderByNameAsc();
}

File: src/main/kotlin/org/example/kalkulationsprogramm/repository/KrankenkasseRepository.kt
package org.example.kalkulationsprogramm.repository

import org.example.kalkulationsprogramm.domain.Krankenkasse
import org.springframework.data.jpa.repository.JpaRepository

interface KrankenkasseRepository : JpaRepository<Krankenkasse, Long> {
    fun findAllByOrderByNameAsc(): List<Krankenkasse>

    fun findByAktivTrueOrderByNameAsc(): List<Krankenkasse>
}
